point_pattern.py

```diff
@@ -124,7 +124,7 @@ def check_in(point, point_list):
     point_list : list
                  in the form [point, point_1, point_2, ..., point_n]
     """
-    return None # Add the logic to check if a point is in the point list here
+    return (point in point_list) # Add the logic to check if a point is in the point list here
```

Comment:
You could also return `point in point_list` and remove the redundant `()`.  This is a style thing, not a requirements, but something that a linter will throw a warning about.